packages/messages/__tests__/testUtils.js

```diff
@@ -0,0 +1,57 @@
+const UserChatMessagesMock = [
+  {
+    _id: '1',
+    chatId: '1',
+    text: 'Anim qui quis cillum qui ea mollit sit aliquip laborum consequat voluptate irure velit exercitation.',
+    createdAt: '2022-01-01T00:00:00.000Z',
+    deletedAt: null,
+    createdBy: 'customer-1',
+  },
+  {
+    _id: '2',
+    chatId: '1',
+    text: 'Anim qui quis cillum qui ea mollit sit aliquip laborum consequat voluptate irure velit exercitation.',
+    createdAt: '2022-01-01T00:00:00.000Z',
+    deletedAt: null,
+    createdBy: 'host-1',
+  },
+];
+
+const UserChatsMock = [
+  {
+    _id: '1',
+    bookingId: 'booking-1',
+    hostId: 'host-1',
+    customerId: 'customer-1',
+    createdAt: '2022-01-01T00:00:00.000Z',
+    updatedAt: '2022-01-01T00:00:00.000Z',
+    deletedAt: null,
+  },
+];
+
+const MessageModelMock = {
+  find: () => new Promise((resolve) => { resolve(UserChatMessagesMock); }),
+  findOne: () => new Promise((resolve) => { resolve(UserChatMessagesMock[0]); }),
+  countDocuments: () => new Promise((resolve) => { resolve(UserChatMessagesMock.length); }),
+};
+
+const ChatModelMock = {
+  find: () => new Promise((resolve) => { resolve(UserChatsMock); }),
+  findOne: () => new Promise((resolve) => { resolve(UserChatsMock[0]); }),
+  countDocuments: () => new Promise((resolve) => { resolve(UserChatsMock.length); }),
+};
```

Comment:
@JoseNoriegaa as we are using the injection dependency principle so we can create these mocks using jest and delegating to the test the data that needs to return.
```suggestion
const MessageModelMock = {
  find: jest.fn(),
  findOne: jest.fn(),
  countDocuments: jest.fn(),
};

const ChatModelMock = {
  find: jest.fn(),
  findOne: jest.fn(),
  countDocuments: jest.fn(),
};
```

Reply:
Ok, I'll change the implementation of these mocks